app/site/controllers.py

```diff
@@ -63,7 +63,11 @@ def logout():
       302:
         description: Load the Home Page
     """
-    return render_template("logout.html", title='Home'), 200
+    return render_template("logout.html", title='Logout',
+                           s3_cdn=get_s3_cdn_prefix(),
+                           auth0_client_id=app.config['AUTH0_CLIENT_ID'],
+                           auth0_domain=app.config['AUTH0_DOMAIN']
```

Comment:
@Fak3 my bad. These three was already added in context processor. Removing these.